tests/filecheck/projects/riscv-backend-paper/bottom_up.mlir

```diff
@@ -500,20 +519,30 @@ func.func public @pooling_nchw_sum_d1_s2_3x3(
     %Y: memref<1x1x7x7xf64>
 ) -> () {
     %zero_float = arith.constant 0.0 : f64
-    memref_stream.fill %Y with %zero_float : memref<1x1x7x7xf64>
-    memref_stream.generic {
-      bounds = [#builtin.int<1>, #builtin.int<1>, #builtin.int<7>, #builtin.int<7>, #builtin.int<3>, #builtin.int<3>],
+    linalg.generic {
+        indexing_maps = [
+            affine_map<(d0, d1, d2, d3) -> ()>,
+            affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>
+        ],
+        iterator_types = ["parallel", "parallel", "parallel", "parallel"]
+    } ins(%zero_float : f64) outs(%Y : memref<1x1x7x7xf64>) {
+    ^bb0(%in: f64, %out: f64):
+        linalg.yield %in : f64
+    }
+    %alloc = memref.alloc() {alignment = 64 : i64} : memref<3x3xf32>
```

Comment:
note that these allocs get canonicalized away, as they're used only for their shapes